### ISSUE TITLE
Bulk job of POST requests

### ISSUE THREAD

**Issue by tomerbal** [eBay/parallec#86]
Hi,

Is it possible to perform bulk job of multiple POST requests? While each request has the same URL but different body.

Thanks.

**Comment by jeffpeiyt** [comment]
@tomerbal  yes. can you check on http://www.parallec.io/docs/submit-task/#apis-on-variable-replacement-for-heterogeneous-requests
